contrib/render-a11y-string/render-a11y-string.js

```diff
@@ -655,6 +655,10 @@ const handleObject = (
             break;
         }
 
+        case "internal": {
+            break;
+        }
```

Comment:
In reality we should never have to handle this case since all all internal nodes in the AST should get expanded, right?

Reply:
I believe the `\def` function returns an `internal` node (at least that's what the PR main text says), and this throws it away.